README.md

```diff
@@ -200,6 +200,22 @@ translationCount and translationParameters can be combined.
 </table>
 ```
 
+### Adding translation files
+You can also override translations by creating translation file in the following name format: `domain.locale.format`.
+So, to create translation file for this bundle you need to create for instance `KnpPaginatorBundle.tr.yaml` file under `yourpojectroot/translations/` 
```

Comment:
please fix syntax

Reply:
Sorry, which syntax needed fixing, I could not detect any syntax errors

Reply:
"yourpojectroot"